bin/zkServer.sh

```diff
@@ -202,7 +190,7 @@ start-foreground)
     "${ZOO_CMD[@]}" $ZOO_DATADIR_AUTOCREATE "-Dzookeeper.log.dir=${ZOO_LOG_DIR}" \
     "-Dzookeeper.log.file=${ZOO_LOG_FILE}" "-Dzookeeper.root.logger=${ZOO_LOG4J_PROP}" \
     -XX:+HeapDumpOnOutOfMemoryError -XX:OnOutOfMemoryError='kill -9 %p' \
-    -cp "$CLASSPATH" $JVMFLAGS $ZOOMAIN "$ZOOCFG"
+    -cp "$CLASSPATH" "$JVMFLAGS" $ZOOMAIN "$ZOOCFG"
```

Comment:
Another intentionally unquoted `$JVMFLAGS`

```suggestion
    -cp "$CLASSPATH" $JVMFLAGS $ZOOMAIN "$ZOOCFG"
```